tests/loggers/test_mlflow_logger.py

```diff
@@ -431,14 +431,17 @@ def test_mlflow_register_uc_error(tmp_path, monkeypatch):
 
 @device('cpu')
 def test_mlflow_logging_works(tmp_path, device):
-    pytest.importorskip('mlflow')
+    mlflow = pytest.importorskip('mlflow')
 
     mlflow_uri = tmp_path / Path('my-test-mlflow-uri')
     experiment_name = 'mlflow_logging_test'
     test_mlflow_logger = MLFlowLogger(
         tracking_uri=mlflow_uri,
         experiment_name=experiment_name,
+        log_system_metrics=True,
```

Comment:
this is the default so you don't need to set it, right?

Reply:
yea, I would prefer to explicitly set the arg so that the test is robust against changes and easier to read.